04/app.js

```diff
@@ -0,0 +1,28 @@
+function createArray(min, max) {
+  if (typeof min !== "number" || typeof max !== "number")
```

Comment:
Warto pamiętać, że `NaN` to też "number".  Ale `NaN === NaN` => false - można to wykorzystać ;)